benchmarks/Riok.Mapperly.Benchmarks/MappingBenchmarks.cs

```diff
@@ -1,4 +1,4 @@
-﻿using BenchmarkDotNet.Attributes;
```

Comment:
Why is the first import for each file modified? 😁

Reply:
I applied `dotnet format` as requested, and these files were UTF-8-BOM. `dotnet format` changed them to UTF-8. Since BOM is several bytes at the beginning of each file it seems like a line change.

Reply:
Could you undo these changes (adding the UTF-8-BOM)? They make it super hard to review, if we want to add it, we should do such mass changes in a separate PR. Also if I'm running `dotnet format` on the current main branch I don't get any changes 🤔  Which version of `dotnet format` are you using (`dotnet format --version`)?

Reply:
However, the current main definitely has mixed UTF-8 (with BOM and without BOM). We should unify this, would be happy to accept a separate PR for this. Probably we should also invest why the editorconfig / linting didn't work properly here.

Reply:
Hi,
my dotnet format:
```sh
> dotnet format --version
8.0.447701+0cf0f51b02d759e89f2acc09663493ab422548b0
```
I dropped the code style commit, I will create a separate PR.

Reply:
My git is configured to `autocrlf = input`, if configured differently it may hide issues with platform line endings/file encoding

Reply:
PR: #869